docs/source/api/v4/logs.rst

```diff
@@ -92,5 +95,8 @@ Response Structure
 			"user": "admin",
 			"id": 443,
 			"message": "1 delivery services were assigned to test"
-		}
-	]}
+		}],
+		"summary": {
+      "count": 20
+    }
+	}
```

Comment:
You're using space-based indentation here - documentation should use tabs

Reply:
I was using tabs but somehow it doesn't like it. Anyways I updated the docs by copying what was seen in /servers.rst